src/style.css

```diff
@@ -1 +1,88 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+
+.stars {
+  display: flex;
+}
+
+
+/* #region 0star */
+.stars__star {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  flex-shrink: 0;
+}
+
+/* #endregion */
+
+/* #region 1star */
+
+.stars--1 > .stars__star:nth-child(-n+1){
+  background-image: url(images/star-active.svg);
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+/* #endregion */
+
+/* #region 2star */
+
+.stars--2 > .stars__star:nth-child(-n+2){
+  background-image: url(images/star-active.svg);
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+/* #endregion */
+
+/* #region 3star */
+
+.stars--3 > .stars__star:nth-child(-n+3){
+  background-image: url(images/star-active.svg);
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+/* #endregion */
+
+/* #region 4star */
+
+.stars--4 > .stars__star:nth-child(-n+4){
+  background-image: url(images/star-active.svg);
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+/* #endregion */
+
+/* #region 5star */
+
+.stars--5 > .stars__star:nth-child(-n+5){
+  background-image: url(images/star-active.svg);
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+/* #endregion */
```

Comment:
1. Check how we can combine these selectors (try to use `,`)

2. I think we already apply these styles to `.stars__star`, so let's remove it here: 
```css
  height: 16px;
  width: 16px;
  background-position: center;
  background-repeat: no-repeat;
  margin-right: 4px;
  ```